ci/github_status.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+
+import os
+import requests
+import sys
+
+# Map from jenkins status to Github status
+jenkins_github = {
+        'PENDING': 'pending',
+        'SUCCESS': 'success',
+        'null': 'success',
+        'UNSTABLE': 'failure',
+        'FAILURE': 'failure'}
+
+
+def main(gh_user, gh_password, commit, status, target_url, context):
+    uri = 'https://api.github.com/repos/mesosphere/marathon/statuses/{}'.format(commit)
```

Comment:
I actually wanted to ask you today during our 1:1 so we can still do it then but why don't we use github statuses also for regular PR builds? I find it the more "native" way in GitHub.

It is just a general question, not something against this PR...

Reply:
We do use Github statuses. The should show up for each build. However, I've added the bot notifications to transfer more information.